README.md

```diff
@@ -6,7 +6,7 @@ Easily organize windows without using a mouse.
 
 ## System requirements
 
-Spectacle [0.8.6][1] is the last version to support Mac OS X 10.7; future releases will only support 10.9 or greater. Folks wishing to stay on 10.7 should download [0.8.6][1]. Those looking for a version of Spectacle that supports 10.6 can still download [0.7][2].
+Spectacle [1.0][1] is the last version to support Mac OS X 10.7; future releases will only support 10.9 or greater. Folks wishing to stay on 10.7 should download [0.8.6][1]. Those looking for a version of Spectacle that supports 10.6 can still download [0.7][2].
```

Comment:
I see.. I read Spectacle [0.8.6][1] is the last version
This looks a bit misleading imo. A table would be better :)